infra/datalake/networking_stack.py

```diff
@@ -0,0 +1,37 @@
+"""
+Data Lake networking stack.
+"""
+from aws_cdk import aws_ec2, core
+from aws_cdk.core import Tags
+
+
+class NetworkingStack(core.Stack):
+    """Data Lake networking stack definition."""
+
+    # pylint: disable=redefined-builtin,too-many-locals
+    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
```

Comment:
We should not redefine builtins. A common workaround is basically Hungarian notation: add the context to the name, such as `stack_id`.

Reply:
This code was created by CDK at project initialization and I see most of examples on web using the same.

Reply:
That's fine, it just means CDK is doing something bad (IMO) in this specific case. We should still change it.

Reply:
Should be fixed in 767304e77510a3f0271ada38e542c66dac0007ed